__tests__/reactTags.test.js

```diff
@@ -538,10 +544,10 @@ describe('Test ReactTags', () => {
       );
       const $input = $el.find('.ReactTags__tagInputField');
 
-      $input.simulate('change', { target: { value: 'Ea' } });
+      $input.simulate('change', { target: { value: 'Or' } });
```

Comment:
'Or' still here because test message 'handles addition when using default suggestions filter' mean there have to be query with result of zero suggestions to add new tag, as i understand.